### ISSUE TITLE
Should we perform softmax before computing the prediction difference?

### ISSUE THREAD

**Issue by LinkToPast1990** [INK-USC/hierarchical-explanation#3]
```
logits_enb = self.model(inp_enb, segment_ids[:, :inp_enb.size(1)], inp_mask_enb)
logits_ex = self.model(inp_ex, segment_ids[:, :inp_ex.size(1)], inp_mask_ex)
contrib_logits = logits_enb - logits_ex  # [1 * C]
```
Hi, the logits_enb and logits_ex may have very different values, while softmax(logits_enb) and softmax(logits_ex) remains similar. 
Should we perform softmax before computing the prediction difference?

**Comment by LinkToPast1990** [comment]
![image](https://user-images.githubusercontent.com/13929748/92433954-14f0b680-f1d1-11ea-9821-d248cd1a740c.png)
https://arxiv.org/abs/1904.11829
It seems both are okay.
